editor/doc_tools.cpp

```diff
@@ -494,7 +494,7 @@ void DocTools::generate(BitField<GenerateFlags> p_flags) {
 				}
 
 				if (properties_from_instance) {
-					if (E.name == "resource_local_to_scene" || E.name == "resource_name" || E.name == "resource_path" || E.name == "script" || E.name == "resource_scene_unique_id") {
```

Comment:
This has me question if something can be done for these as well to completely get rid of the hardcoded exceptions.

Reply:
Perhaps this is because the `Object` class does not have its own category, so determining its properties/methods requires some tricks. But this needs additional research.